apps/docs/src/basics/breakpoints-grid.md

```diff
@@ -1,3 +1,201 @@
-# Breakpoints and Grid
+# Breakpoints and grid
+
+Column grids in ui design and development establish a structured layout, organizing content, components and elements cohesively. They provide a framework for alignment and distribution, ensuring consistency and hierarchy. Additionally, grids serve as the foundation for smooth responsive behavior, allowing seamless adoption to different screen sizes and resolutions, ultimately enhancing usability and overall user experience.
+
+To see the grid and breakpoints in action, feel free to have a look on the [grid playground](/exapmle).
+
+## Understanding the grid
+
+To understand the anatomy of a column grid, it is essential to comprehend its key components: margins, columns and gutters, which function harmoniously together. Visual examples and further details can be found below.
+
+![description of grid units](/assets/grid-explanation.png)
```

Comment:
We want to focus on accessibility and should try to also extend this to our docs :) 
In the description we should try to explain concepts that are only shown there or point to the in-text explanation:

```suggestion
![Description of how the grid key components are layed out. Multiple vertical tracks are stretching over the whole page: the outermost tracks are "margins". Inbetween is an odd amount of columns, where every even column is a "gutter" and every odd track is a "column"](/assets/grid-explanation.png)
```